core/modules/compare/store/index.ts

```diff
@@ -2,10 +2,9 @@ import { Module } from 'vuex'
 import actions from './actions'
 import getters from './getters'
 import mutations from './mutations'
-import RootState from '../../types/RootState'
-import CompareState from './types/CompareState'
+import CompareState from '../types/CompareState'
 
-const compare: Module<CompareState, RootState> = {
+export const module: Module<CompareState, any> = {
```

Comment:
you can use `RootState` instead of any
```suggestion
export const module: Module<CompareState, RootState> = {
```

Reply:
done